### PR TITLE
:boom: Implement update timeslot feature for candidate and coordinator

### PR DESCRIPTION
**Pull request by Ewentualnie**
add possibility update timeslot:
`POST "/interviewers/{interviewerId}/slots/{slotId}"`

### CODE REVIEW

**Comment by Mo-og** [comment]
|File|Coverage [14.17%]|:green_apple:|
|:-|:-:|:-:|
|[SlotController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/f6b841bd25a1b9be9b7741dd71b3580fb810d808/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FSlotController.java)|70.83%|:green_apple:|
|[InterviewerService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/f6b841bd25a1b9be9b7741dd71b3580fb810d808/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|0%|:x:|

|Total Project Coverage|33.31%|:green_apple:|
|:-|:-:|:-:|

**Comment by Mo-og** [comment]
|File|Coverage [8.85%]|:x:|
|:-|:-:|:-:|
|[SlotController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/ae5812b61dc81b5bec704948afd91e032c4e1cd0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FSlotController.java)|60.61%|:green_apple:|
|[CandidateService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/ae5812b61dc81b5bec704948afd91e032c4e1cd0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FCandidateService.java)|0%|:x:|
|[InterviewerService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/ae5812b61dc81b5bec704948afd91e032c4e1cd0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|0%|:x:|

|Total Project Coverage|33.31%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/services/CandidateService.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public CandidateTimeSlot updateSlot(long candidateId, CandidateTimeSlot slot) {
    * @param id candidate id to look for
    * @return candidate stored by given id
    */
-  public User getById(Long id) {
+  public User getUserById(Long id) {
```
Comment (Mo-og) [inline src/main/java/com/intellias/intellistart/interviewplanning/services/CandidateService.java:104]: Should not be renamed. It is better to preserve same method namings for service interfaces

File: src/main/java/com/intellias/intellistart/interviewplanning/services/InterviewerService.java [hunk 2]
```diff
@@ -79,20 +78,21 @@ public Set<InterviewerTimeSlot> getRelevantInterviewerSlots(Long interviewerId)
...
-   * @param from    start time
-   * @param to      end time
-   * @param day     day of week
-   * @param weekNum number of week
+   * @param slotId              slot id
+   * @param interviewerTimeSlot slot
+   * @return updated slot
    */
-  public InterviewerTimeSlot updateSlot(long id, String from, String to, String day, int weekNum) {
+  public InterviewerTimeSlot updateSlot(long interviewerId, long slotId,
```
Comment (Mo-og) [inline src/main/java/com/intellias/intellistart/interviewplanning/services/InterviewerService.java:85]: long -> Long

**Review by Mo-og — CHANGES_REQUESTED** [review]
minor code cleanup

**Comment by Mo-og** [comment]
|File|Coverage [8.97%]|:x:|
|:-|:-:|:-:|
|[SlotController.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/f79b9e76199769365c3191f1dec14c52ba713154/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FSlotController.java)|58.82%|:green_apple:|
|[CandidateService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/f79b9e76199769365c3191f1dec14c52ba713154/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FCandidateService.java)|0%|:x:|
|[InterviewerService.java](https://github.com/Mo-og/intellistart-java-2022-debuggers/blob/f79b9e76199769365c3191f1dec14c52ba713154/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FInterviewerService.java)|0%|:x:|

|Total Project Coverage|33.37%|:green_apple:|
|:-|:-:|:-:|

**Comment by Ewentualnie** [comment]
add changes

**Review by Mo-og — APPROVED** [review]

**Pull request merged**